tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/InstanceMethods.java

```diff
@@ -151,6 +151,22 @@ public String unionWithErrorReturnHandle() throws InterruptedException {
         return "handle ret";
     }
 
+    public byte[] unionWithErrorReturnByteArray() throws InterruptedException {
+        byte[] byteArray = "Byte Test".getBytes();
+        if (false) {
+            throw new InterruptedException();
+        }
+        return byteArray;
+    }
+
+    public String[] unionWithErrorReturnStringArray() throws InterruptedException {
```

Comment:
Let's remove the duplicate test since we do not need the same.

Reply:
Actually the return type of the extern function is `any|error`. Changed the function name. Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/31288/commits/d3a4640e53d1125607a5898e6dc0ffdf2b49cb1d